ocaml/testsuite/tests/typing-layouts-float64/alloc.ml

```diff
@@ -167,3 +167,18 @@ let _ =
   let _ = measure_alloc_value (fun () -> cse_test false r) in
   let allocs = get_exact_allocations () in
   Printf.printf "CSE test (0 bytes):\n  allocated bytes: %.2f\n" allocs
+
+let[@inline never] literal_test () =
+  let open Float_u in
+  (#1. + #2.) * (#3. - #4.) / (#3. ** #1.)
```

Comment:
I would be more convinced about zero-allocness if some of these floats were passed as args.

Reply:
Made that change and it now passes in a few unboxed floats as arguments